tracee/tracee_test.go

```diff
@@ -18,69 +18,79 @@ func TestReadArgFromBuff(t *testing.T) {
 		{
 			name: "intT",
 			input: []byte{1, //intT
+				0,                      //TODO proper tag?
 				0xFF, 0xFF, 0xFF, 0xFF, //-1
 			},
 			expectedArg: int32(-1),
 		},
 		{
 			name: "uintT",
 			input: []byte{2, //uintT
+				0,                      //TODO proper tag?
 				0xFF, 0xFF, 0xFF, 0xFF, //4294967295
 			},
 			expectedArg: uint32(4294967295),
 		},
 		{
 			name: "longT",
 			input: []byte{3, //longT
+				0,                                              //TODO proper tag?
 				0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, //-1
 			},
 			expectedArg: int64(-1),
 		},
 		{
 			name: "ulongT",
 			input: []byte{4, //ulongT
+				0,                                              //TODO proper tag?
 				0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, //18446744073709551615
 			},
 			expectedArg: uint64(18446744073709551615),
 		},
 		{
 			name: "offT",
 			input: []byte{5, //offT
+				0,                                              //TODO proper tag?
 				0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, //18446744073709551615
 			},
 			expectedArg: uint64(18446744073709551615),
 		},
-		{
+		{ //TODO modeT might need different treatment
 			name: "modeT",
 			input: []byte{6, //modeT
+				0,                    //TODO proper tag?
 				0xB6, 0x11, 0x0, 0x0, //0x000011B6 == 010666 == S_IFIFO|S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP|S_IROTH|S_IWOTH
 			},
-			expectedArg: "S_IFIFO|S_IRUSR|S_IWUSR|S_IRGRP|S_IWGRP|S_IROTH|S_IWOTH",
+			expectedArg: uint32(0x000011B6),
```

Comment:
`modeT` is currently grouped together with `uinT` and `devT`, not sure if this is intended base on existing test.